### ISSUE TITLE
`block in diff': no implicit conversion of GraphQL::SchemaComparator::Changes::EnumValueDescriptionChanged into Array (TypeError)

### ISSUE THREAD

**Issue by arquio** [xuorig/graphql-schema_comparator#9]
Hi @xuorig !!

First thanks you for this project!

Now I try compare a more complex schemas and returns this error:

```
graphql-schema compare gw20171103.graphql gw20171108.graphql 
🤖  Checking for changes...
/home/dcalle/.gem/ruby/2.4.0/gems/graphql-schema_comparator-0.3.1/lib/graphql/schema_comparator/diff/enum.rb:23:in `block in diff': no implicit conversion of GraphQL::SchemaComparator::Changes::EnumValueDescriptionChanged into Array (TypeError)
	from /home/dcalle/.gem/ruby/2.4.0/gems/graphql-schema_comparator-0.3.1/lib/graphql/schema_comparator/diff/enum.rb:44:in `block in each_common_value'
	from /home/dcalle/.gem/ruby/2.4.0/gems/graphql-schema_comparator-0.3.1/lib/graphql/schema_comparator/diff/enum.rb:40:in `each'
	from /home/dcalle/.gem/ruby/2.4.0/gems/graphql-schema_comparator-0.3.1/lib/graphql/schema_comparator/diff/enum.rb:40:in `each_common_value'
	from /home/dcalle/.gem/ruby/2.4.0/gems/graphql-schema_comparator-0.3.1/lib/graphql/schema_comparator/diff/enum.rb:19:in `diff'
	from /home/dcalle/.gem/ruby/2.4.0/gems/graphql-schema_comparator-0.3.1/lib/graphql/schema_comparator/diff/schema.rb:45:in `changes_in_type'
	from /home/dcalle/.gem/ruby/2.4.0/gems/graphql-schema_comparator-0.3.1/lib/graphql/schema_comparator/diff/schema.rb:25:in `block in diff'
	from /home/dcalle/.gem/ruby/2.4.0/gems/graphql-schema_comparator-0.3.1/lib/graphql/schema_comparator/diff/schema.rb:103:in `block in each_common_type'
	from /home/dcalle/.gem/ruby/2.4.0/gems/graphql-schema_comparator-0.3.1/lib/graphql/schema_comparator/diff/schema.rb:99:in `each'
	from /home/dcalle/.gem/ruby/2.4.0/gems/graphql-schema_comparator-0.3.1/lib/graphql/schema_comparator/diff/schema.rb:99:in `each_common_type'
	from /home/dcalle/.gem/ruby/2.4.0/gems/graphql-schema_comparator-0.3.1/lib/graphql/schema_comparator/diff/schema.rb:24:in `diff'
	from /home/dcalle/.gem/ruby/2.4.0/gems/graphql-schema_comparator-0.3.1/lib/graphql/schema_comparator.rb:25:in `compare'
	from /home/dcalle/.gem/ruby/2.4.0/gems/graphql-schema_comparator-0.3.1/bin/graphql-schema:15:in `compare'
	from /home/dcalle/.gem/ruby/2.4.0/gems/thor-0.20.0/lib/thor/command.rb:27:in `run'
	from /home/dcalle/.gem/ruby/2.4.0/gems/thor-0.20.0/lib/thor/invocation.rb:126:in `invoke_command'
	from /home/dcalle/.gem/ruby/2.4.0/gems/thor-0.20.0/lib/thor.rb:387:in `dispatch'
	from /home/dcalle/.gem/ruby/2.4.0/gems/thor-0.20.0/lib/thor/base.rb:466:in `start'
	from /home/dcalle/.gem/ruby/2.4.0/gems/graphql-schema_comparator-0.3.1/bin/graphql-schema:53:in `<top (required)>'
	from /usr/local/bin/graphql-schema:23:in `load'
	from /usr/local/bin/graphql-schema:23:in `<main>'
```

Is possible it is a an error caused by enum type?

[gw20171103.graphql.txt](https://github.com/xuorig/graphql-schema_comparator/files/1470364/gw20171103.graphql.txt)

[gw20171108.graphql.txt](https://github.com/xuorig/graphql-schema_comparator/files/1470362/gw20171108.graphql.txt)

Thanks a lot!!

**Comment by xuorig** [comment]
Thanks @arquio will look into it 👁

**Comment by xuorig** [comment]
Fixed in `0.3.2` @arquio 

Tested against your changes:

[results.txt](https://github.com/xuorig/graphql-schema_comparator/files/1472771/results.txt)

**Comment by arquio** [comment]
Now, Perfect!!!!

Great tool!! Great work!

Thanks a lot!!!
